index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.3.0/css/all.min.css"
+      integrity="sha512-SzlrxWUlpfuzQ+pcUCosxcglQRNAq/DZjVsC0lE40xsADsfeQoEypE+enwcOiGjk/bSuGGKHEyjSoQ1zVisanQ=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.css"
+      integrity="sha512-5A8nwdMOWrSz20fDsjczgUidUBR8liPYU+WymTZP1lmY9G6Oc7HlZv156XqnsgNUzTyMefFTcsFH/tnJE/+xBg=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <section class="header">
+      <header>
+        <div class="logo"><a href="#">My Logo</a></div>
+
+        <i class="fa fa-bars" aria-hidden="true"></i>
+        <div class="nav-links">
+          <a href="#">portfolio</a>
+          <a href="#">about</a>
+          <a href="#">contact</a>
+        </div>
+      </header>
+      <div class="headline">
+        <h1 class="headline-alexandra_small">
+          I’m Alexandra. Glad to see you!
+        </h1>
+        <h1 class="headline-alexandra_big">
+          I’m Alexandra.<br />
+          Glad to see you!
+        </h1>
+        <p class="description">
+          I’m a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hesitate to contact
+          me.
+        </p>
+        <div class="connect"><a href=""> LETS CONNECT</a></div>
+        <div class="icons">
+          <a href=""><i class="fa fa-twitter" aria-hidden="true"></i></a>
+          <a href=""><i class="fa fa-linkedin" aria-hidden="true"></i></a>
+          <a href=""><img src="images/Vector.png" alt="medium-icon" /></a>
+          <a href=""><i class="fa fa-github" aria-hidden="true"></i></a>
+          <a href=""><i class="fa fa-hand-peace-o" aria-hidden="true"></i></a>
+        </div>
```

Comment:
- [ ] It would be better for the list of icons to use a ```ul``` tag to group them, and ```li``` per icon; 🙏 
_using the ul and li tags in HTML can improve the organization, structure, styling, and semantics of your content, making it more accessible, user-friendly, and maintainable._